src/plugins/visualize/public/application/utils/get_table_columns.tsx

```diff
@@ -38,6 +38,8 @@ import { FormattedMessage } from '@osd/i18n/react';
 
 import { ApplicationStart } from 'opensearch-dashboards/public';
 import { VisualizationListItem } from 'src/plugins/visualizations/public';
+import moment from 'moment';
+import { IUiSettingsClient } from '../../../../../core/public';
```

Comment:
minor: are we able to import from `'src/core/public'`?

Reply:
We have "IUiSettingsClient" as a public export inside src/core/public/index.ts, so yes.

Reply:
Is this still going to be addressed?

Reply:
My apologies if it wasn't clear, I just meant changing the import to `from 'src/core/public'`